workshops/README.md

```diff
@@ -33,6 +34,6 @@ community workshops that have been created so far are listed below in
 alphabetical order.
 
 | Workshop                           | What You Build                                                 | Author                                 |
-| ---------------------------------- | -------------------------------------------------------------- | -------------------------------------- |
+|:-----------------------------------|:---------------------------------------------------------------|:---------------------------------------|
```

Comment:
Indeed! Since all of our tables don't currently have these extra colons, let's remove them for now.

I'll make sure these get into our Markdown standards.